skt/reporter.py

```diff
@@ -420,13 +427,17 @@ def getreport(self):
         msg = ['Hello,\n',
                'We appreciate your contributions to the Linux kernel and '
                'would like to help',
-               'test them. Below are the results of automatic tests we ran on '
-               'a patchset',
-               'you\'re involved with, with hope it will help you find '
-               'possible issues sooner.',
-               '\n']
+               'test them. Below are the results of automatic tests we ran']
+        if self.mergedata['localpatch'] or self.mergedata['patchwork']:
+            msg[-1] += ' on a patchset'
+            msg += ['you\'re involved with, with hope it will help you find '
```

Comment:
This wording is okay, but we could say something like `automatic tests we ran on a patchset that involves you`. I was trying to find a way to avoid using the contraction (`you're`) since that can be challenging for some non-native English speakers.

Reply:
This was what @spbnick came up with after our long discussion about the intro message and I kept the wording here. We can discuss this all together if you think the wording should be changed a bit.